### PR TITLE
Added new plugins and removed a setup.

### PR DESCRIPTION
**Pull request by OPADILOP**
Removed StudioBeam Development's setup because it's no longer available. Added IamTheDefender's Cosmetics and Item Rotation plugin. Added NotLew_x's AdventureMode plugin. Added RRS's Discord Stats plugin. Added "by" in some lines.

### CODE REVIEW

File: docs/BedWars1058/addons.md [hunk 1]
```diff
@@ -12,56 +12,68 @@ To add your own addon(s), use the edit button at the bottom of the page, click t
...
-- [Spectator & Play-Again Addon {Menu}](https://polymart.org/resource/1937/) - by StudioBeam Development
-- [Arena Start Message - Allows you to send Arena Join Notification to the players in lobby](https://polymart.org/resource/1880/) - by StudioBeam Development
 - [Cosmetics - 300+ In-built cosmetics, custom cosmetics, custom rarities, multifunctional, packet-based and more](https://polymart.org/resource/1619/) - by Mher
 - [Private Games - Create private games and play with friends](https://polymart.org/resource/1620/) - by Mher
+- [LuckyBlock NTD - Add LuckyBlocks to your ingame spawners](https://www.spigotmc.org/resources/94872/) - by danirod12
+- [CentralBot - A Discord Stats Plugin which displays players info from BedWars1058!](https://www.spigotmc.org/resources/107498/) - by RRS
+- [DiscordStats for BedWars1058 - Get the stats of your players on discord!](https://polymart.org/resource/1889/) - by STN-Studios
+- [Quests - A powerful quest system based for BedWars1058.](https://polymart.org/resource/1802) - by reussy
+- [Spectator Options - Add useful settings for spectators!](https://polymart.org/resource/1916/) - by CubeCrafter72
```
Comment (andrei1058) [inline docs/BedWars1058/addons.md:24]: This is no longer available..

**Review by andrei1058 — CHANGES_REQUESTED** [review]

**Comment by tomkeuper** [comment]
Please resolve changes so it can be merged

**Review by OPADILOP — COMMENTED** [review]
done as you said

**Comment by andrei1058** [comment]
sorry but there's a new conflict :((

**Comment by OPADILOP** [comment]
No problem fixing it
Now

**Comment by OPADILOP** [comment]
I have fixed and removed the plugins which were added. Because as my rearrangement, the plugins were already added to the upwards.

**Review by andrei1058 — APPROVED** [review]

**Pull request merged**